avaframe/out3SimpPlot/plotSettings.py

```diff
@@ -14,7 +14,7 @@
 # define figure dimentions
 figW = 6
 figH = 6
-# define figure resolution
+# define figure resolution (dpi)
```

Comment:
Should this and the safefig.format end up in the general .ini?
I think it would make sense to put it there..

Reply:
I could move it there yes.  But it is harder to all then no?